backend-models/service-tests/courseedservice-tests.txt

```diff
@@ -0,0 +1,258 @@
+
+courseedservice.php
+
+---------------------------------------------------------------------------------------------------------------------
+- Insert into course (line 79)
+---------------------------------------------------------------------------------------------------------------------
+
+*********
+* pre-req:
+********* 
+
+{(checklogin) && ($ha)} IS TRUE
+
+*********
+* send parameters inputs to courseedservice.php 
+*********
+send
+{
+    $opt = "NEW";
+    $userid = "UNK"; or $userid = !NULL;
+    $coursecode = "G1337";
```

Comment:
This value cannot be used. I receive this error when i try to make an insert.
![image](https://user-images.githubusercontent.com/102600391/235891309-ffbb784a-8876-4917-ab9b-5dbc616306b1.png)

The entered value has to be unique in the table "course", this is indicated by this line in mySQL.
![image](https://user-images.githubusercontent.com/102600391/235891135-06b17254-431d-44ae-a2b5-f8fe5ab9490c.png)
In other words you need to make sure that the value does not already exist in the table.